### PR TITLE
Link launch_pool and xprism_boost

### PR DESCRIPTION
**Pull request by HoldMyBeer1111**

### CODE REVIEW

File: src/integration_tests/deploy_prism/definition.ts [hunk 1]
```diff
@@ -259,17 +270,14 @@ async function prism_init_verbose(
...
 
-	// instanstiate boost contract
-	let prism_xprism_boost_addr = await init_prism_xprism_boost(
-		lcd_client,
-		sender,
-		prism_governance_info.xprism_token_addr
-	)
-	console.log(`prism_xprism_boost instantiated\n\taddress: ${prism_xprism_boost_addr}`);
+	const msg_update_config = { update_config: { launch_pool_contract: prism_launch_pool_addr } };
+	await execute_contract(lcd_client, sender, prism_xprism_boost_addr, msg_update_config);
```
Comment (steven4354) [inline src/integration_tests/deploy_prism/definition.ts:279]: @HoldMyBeer1111 can we move this contract call to inside `init_prism_xprism_boost` function if it needs this msg to launch? just to keep it together into 1 function

Comment (HoldMyBeer1111) [reply src/integration_tests/deploy_prism/definition.ts:279]: guess no, we have to instantiate launch_pool at first, which already requires xprism_boost address

Comment (steven4354) [reply src/integration_tests/deploy_prism/definition.ts:279]: gotcha sg, the pr looks good

**Comment by steven4354** [comment]
just going to merge this in and update my pr with your changes

**Pull request merged**